README.md

```diff
@@ -24,6 +24,18 @@ optional arguments:
   -v VERSION, --version VERSION
                         Version (default=latest)
 ```
+Example:
+```bash
+# cd artifacthub
```

Comment:
I think it's considered best practice that this file lives within this repo, and therefore should be executed from the root-folder of this repo. I agree with @Raffy23 that it could be mentioned in the paragraph before though. Defintely not worth a `cd artifacthub` though.